app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserActivity.kt

```diff
@@ -36,7 +37,12 @@ open class ExternalAppBrowserActivity : HomeActivity() {
     final override fun getIntentSessionId(intent: SafeIntent) = intent.getSessionId()
 
     override fun startupTelemetryOnCreateCalled(safeIntent: SafeIntent, hasSavedInstanceState: Boolean) {
-        components.appStartupTelemetry.onExternalAppBrowserOnCreate(safeIntent, hasSavedInstanceState)
+        components.appStartupTelemetry.onExternalAppBrowserOnCreate(
+            safeIntent,
+            hasSavedInstanceState,
+            homeActivityInitTimeStampNanoSeconds,
```

Comment:
Surprisingly to me, `HomeActivity.init` is called first. Here's the timings I pulled:
```
External app 619720736139198
HomeActivity 619720735597114
```

The difference is ~0.5ms on the Pixel 2. Sounds like we should keep it as is. I'll comment about this to avoid confusion.

Reply:
Done in the new PR.